src/dmd/traits.d

```diff
@@ -836,6 +836,19 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
         }
         ex = ex.ctfeInterpret();
 
+        bool includeTemplates = false;
+        if (dim == 3 && e.ident == Id.getOverloads)
+        {
+            auto b = isExpression((*e.args)[2]);
+            b = b.ctfeInterpret();
+            if (!b.type.equals(Type.tbool))
+            {
+                e.error("bool expected as third argument of __traits(getOverloads).");
```

Comment:
tick marks around `bool` and `__traits(getOverloads)` for highlighting in the error message.

Reply:
Also, we need a test case for this error message.